### PR TITLE
fix(@desktop): Use colorId to find correct color of users&members

### LINKED ISSUE #6366: Using model.colorId in UserPanel list.
### Description

Fix "// FIXME: use model.colorId" in:
1. UserListPanel
2. CommunityMembersTabPanel (added in #6280)

### PR DESCRIPTION
**Pull request by endulab**
colorId is set in the model instead of using qml function.

Fix #6366

### Affected areas

Users/members colors

### Screenshot of functionality (including design for comparison)

image

🍺

### CODE REVIEW

File: src/app/modules/shared_models/member_item.nim [hunk 1]
```diff
@@ -17,7 +17,7 @@ proc initMemberItem*(
   localNickname: string,
   alias: string,
   icon: string,
-  colorId: int = 0,
+  colorId: int,
```
Comment (osmaczko) [inline src/app/modules/shared_models/member_item.nim:20]: :+1:

File: src/app/modules/shared_models/user_item.nim [hunk 2]
```diff
@@ -83,7 +83,7 @@ proc initUserItem*(
     localNickname: string = "",
     alias: string = "",
     icon: string,
-    colorId: int = 0,
+    colorId: int,
```
Comment (osmaczko) [inline src/app/modules/shared_models/user_item.nim:86]: :+1:

**Review by osmaczko — APPROVED** [review]
Very nice, it was on my todo list for a while already, thanks!

File: ui/app/AppLayouts/Chat/panels/UserListPanel.qml [hunk 3]
```diff
@@ -95,7 +95,7 @@ Item {
             }
             asset.isImage: (asset.name !== "")
             asset.isLetterIdenticon: (asset.name === "")
-            asset.color: Utils.colorForPubkey(model.pubKey) // FIXME: use model.colorId
+            asset.color:  Utils.colorForColorId(model.colorId)
```
Comment (faust4exe) [inline ui/app/AppLayouts/Chat/panels/UserListPanel.qml:98]: Additional space before Utils is not needed

**Review by faust4exe — APPROVED** [review]
Looks good - just one small comment

**Review by micieslak — APPROVED** [review]
lgtm

**Pull request merged**